wiki/en/Include-Shared-Commands.md

```diff
@@ -2,7 +2,7 @@
 [comment]: # (This is an include file for use in multiple documents)
 
 - `-h` or `--help` Display help text
-- `-i` or `--inifile` Set location of initialization file (overrides default)
+- `-i` or `--inifile` Set location of initialization file (overrides default. On macOS you should only give a filename, not a path, as the App is sandboxed. This will save the files to `~/Library/Containers/io.jamulus.Jamulus/Data/`)
```

Comment:
```suggestion
- `-i` or `--inifile` Set location of initialization file (overrides default. On macOS you should only give a filename, not a path, as the App is sandboxed. This will save the files to `~/Library/Containers/io.jamulus.Jamulus/Data/` replace io.jamulus.Jamulus with io.jamulus.JamulusServer for the server)
```

Reply:
Ok. Checked. The folder is correct.